pkgs/servers/web-apps/weblate/default.nix

```
@@ -0,0 +1,66 @@
+{ lib, fetchFromGitHub, poetry2nix, gettext, isocodes, xorg }:
+
+(poetry2nix.mkPoetryApplication {
+  src = fetchFromGitHub {
+    owner = "WeblateOrg";
+    repo = "weblate";
+    rev = "weblate-4.14";
+    sha256 = "sha256-PsFYL/yKu90QQ0/XdbVFHY1uTU/6H+YfitqFb6Ic4E8=";
+  };
+  pyproject = ./pyproject.toml;
+  poetrylock = ./poetry.lock;
+  # The default timeout for the celery check is much too short upstream, so
+  # we increase it. I guess this is due to the fact that we test the setup
+  # very early into the initialization of the server, so the load might be
+  # higher compared to production setups?
+  patches = [ ./longer-celery-wait-time.patch ];
```

Leave space for other patches.

```suggestion
  patches = [
    # The default timeout for the celery check is much too short upstream, so
    # we increase it. I guess this is due to the fact that we test the setup
    # very early into the initialization of the server, so the load might be
    # higher compared to production setups?
    ./longer-celery-wait-time.patch
  ];
```

---

Done.